apps/web/modules/settings/billing/components/BillingCredits.tsx

```diff
@@ -132,117 +169,154 @@ export default function BillingCredits() {
-              <div className="text-subtle">
-                <div>
-                  {t("total_credits", {
-                    totalCredits: creditsData.credits.totalMonthlyCredits,
-                  })}
+        <div className="bg-default border-muted flex w-full rounded-[10px] border px-5 py-4">
+          <div className="w-full">
+            {totalCredits > 0 ? (
+              <>
+                <div className="mb-4">
+                  <CreditRow
+                    label={t("monthly_credits")}
+                    value={creditsData.credits.totalMonthlyCredits ?? 0}
+                    isBold={true}
+                    underline="dashed"
+                  />
+                  <CreditRow label={t("credits_used")} value={totalUsed} underline="solid" />
+                  <CreditRow
+                    label={t("total_credits_remaining")}
+                    value={creditsData.credits.totalRemainingCreditsForMonth}
+                  />
+                  <div className="mt-4">
+                    <ProgressBar color="green" percentageValue={100 - teamCreditsPercentageUsed} />
+                  </div>
+                  {/*750 credits per tip*/}
+                  <div className="mt-4 flex flex-1 items-center justify-between">
+                    <p className="text-subtle text-sm font-medium leading-tight">
+                      {orgSlug ? t("credits_per_tip_org") : t("credits_per_tip_teams")}
+                    </p>
+                    <Button
+                      href={
+                        orgSlug
+                          ? `/settings/organizations/${orgSlug}/members`
+                          : `/settings/teams/${teamId}/members`
+                      }
+                      size="sm"
+                      color="secondary">
+                      {t("add_members_no_ellipsis")}
+                    </Button>
+                  </div>
                 </div>
-                <div>
-                  {t("remaining_credits", {
-                    remainingCredits: creditsData.credits.totalRemainingMonthlyCredits,
-                  })}
+                <div className="-mx-5 mt-5">
+                  <hr className="border-subtle" />
+                </div>
+              </>
+            ) : (
+              <></>
+            )}
+
+            {/*Auto Top-Up goes here when we have it*/}
+            {/*<div className="-mx-5 mt-5">
+              <hr className="border-subtle" />
+            </div>*/}
+            {/*Additional Credits*/}
+            <form onSubmit={handleSubmit(onSubmit)} className="mt-4 flex">
+              <div className="-mb-1 mr-auto w-full">
+                <div className="flex justify-between">
+                  <Label>{t("additional_credits")}</Label>
+                  <div className="flex items-center gap-1">
+                    <p className="text-sm font-semibold leading-none">
+                      <span className="text-subtle font-medium">{t("current_balance")}</span>{" "}
```

Comment:
Current balance should have a little more space below same as Additional Credits
<img width="647" height="72" alt="Screenshot 2025-09-19 at 10 55 27 AM" src="https://github.com/user-attachments/assets/1f2498de-c277-48f7-85d5-44c94dd30eaa" />